mongo/bulk_write_models.go

```diff
@@ -44,21 +50,23 @@ func NewDeleteOneModel() *DeleteOneModel {
 	return &DeleteOneModel{}
 }
 
-// SetFilter sets the filter for the DeleteOneModel.
+// SetFilter specifies a filter to use to select the document to delete. The filter must be a document containing query
+// operators. It cannot be nil. If the filter matches multiple documents, one will be selected from the matched set.
```

Comment:
We tend to avoid talk of "sets" in MongoDB, I think.  Perhaps "from the matching documents"?